persistence/src/main/java/edu/softserve/zoo/persistence/specification/impl/animal/AnimalGetByIdSpecification.java

```diff
@@ -0,0 +1,27 @@
+package edu.softserve.zoo.persistence.specification.impl.animal;
+
+import edu.softserve.zoo.model.Animal;
+import edu.softserve.zoo.persistence.specification.hibernate.CriterionSpecification;
+import org.hibernate.criterion.Criterion;
+import org.hibernate.criterion.Restrictions;
+
+/**
+ * @author Serhii Alekseichenko
+ */
+public class AnimalGetByIdSpecification implements CriterionSpecification<Animal> {
```

Comment:
Maybe) but now we need to wait for hotfix fix with idToRemove)))
On May 12, 2016 08:44, "Ihor Tsvietkov" <notifications@github.com> wrote:

> In
> persistence/src/main/java/edu/softserve/zoo/persistence/specification/impl/animal/AnimalGetByIdSpecification.java
> <https://github.com/ITsvetkoFF/Kv-014/pull/77#discussion_r62967450>:
>
> > @@ -0,0 +1,27 @@
> > +package edu.softserve.zoo.persistence.specification.impl.animal;
> > +
> > +import edu.softserve.zoo.model.Animal;
> > +import edu.softserve.zoo.persistence.specification.hibernate.CriterionSpecification;
> > +import org.hibernate.criterion.Criterion;
> > +import org.hibernate.criterion.Restrictions;
> > +
> > +/**
> > + * @author Serhii Alekseichenko
> > + */
> > +public class AnimalGetByIdSpecification implements CriterionSpecification<Animal> {
>
> Might be generic one after andrew's hotfix merge... ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ITsvetkoFF/Kv-014/pull/77/files/048cb59950cd248c2fd60e73aceb48c5b396d668#r62967450>
>